src/domains/idpay-app/99_data_idpay_common.tf

```diff
@@ -8,3 +8,9 @@ data "azurerm_eventhub" "eventhub_idpay_reward_notification_storage_events" {
   namespace_name      = local.idpay_eventhubs.evh01.namespace
   resource_group_name = local.idpay_eventhubs.evh01.resource_group_name
 }
+
+data "azurerm_api_management_product" "mock_api_product" {
```

Comment:
This data exists only on DEV and UAT, while PROD doesn't contains mock api product. Can you use a count to conditionally get them?

Reply:
Variables added